images/linux/scripts/installers/java-tools.sh

```diff
@@ -26,17 +30,20 @@ fi
 if isUbuntu20 ; then
     DEFAULT_JDK_VERSION=11
     defaultLabel11="(default)"
+    apt-get update
 fi
 
-apt-get -y install zulu-8-azure-jdk=\*
-apt-get -y install zulu-11-azure-jdk=\*
-apt-get -y install zulu-12-azure-jdk=\*
-update-java-alternatives -s /usr/lib/jvm/zulu-8-azure-amd64
+# Install only LTS versions.
+apt-get -y install adoptopenjdk-8-hotspot=\*
+apt-get -y install adoptopenjdk-11-hotspot=\*
 
-echo "JAVA_HOME_8_X64=/usr/lib/jvm/zulu-8-azure-amd64" | tee -a /etc/environment
-echo "JAVA_HOME_11_X64=/usr/lib/jvm/zulu-11-azure-amd64" | tee -a /etc/environment
-echo "JAVA_HOME_12_X64=/usr/lib/jvm/zulu-12-azure-amd64" | tee -a /etc/environment
-echo "JAVA_HOME=/usr/lib/jvm/zulu-${DEFAULT_JDK_VERSION}-azure-amd64" | tee -a /etc/environment
+# Set Default Java version.
+update-java-alternatives -s /usr/lib/jvm/adoptopenjdk-${DEFAULT_JDK_VERSION}-hotspot-amd64
+
+echo "JAVA_HOME_8_X64=/usr/lib/jvm/adoptopenjdk-8-hotspot-amd64" | tee -a /etc/environment
+echo "JAVA_HOME_11_X64=/usr/lib/jvm/adoptopenjdk-11-hotspot-amd64" | tee -a /etc/environment
+echo "JAVA_HOME_12_X64=/usr/lib/jvm/adoptopenjdk-12-hotspot-amd64" | tee -a /etc/environment
```

Comment:
Looks like `JAVA_HOME_12_X64` variable will be created on Ubuntu 20? Do we need it?

Reply:
Thank you, Fixed.